ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -100,6 +100,17 @@ - (void)reload
   }
 }
 
+- (void)reloadFromOrigin
```

Comment:
Sorry, I reffered the `reload` function.

But this special logic may be inappropriate.